addons/turbowarp-redirect/addon.json

```
@@ -0,0 +1,38 @@
+{
+  "name": "Turbowarp redirect",
+  "description": "Redirects unshared projects to Turbowarp.",
+  "tags": ["community"],
+  "credits": [
+    {
+      "name": "illogicalapple",
+      "link": "https://scratch.mit.edu/users/illogicalapple/"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["https://scratch.mit.edu/projects/*"]
+    }
+  ],
+  "settings": [
+  	{
+      "name": "Redirect to",
+      "id": "redirectUrl",
+      "type": "select",
+      "default": "//turbowarp.org/",
+      "potentialValues": [
+        {
+          "id": "//turbowarp.org/",
+          "name": "Turbowarp"
```

```suggestion
  "description": "Redirects unshared projects to TurboWarp.",
  "tags": ["community"],
  "credits": [
    {
      "name": "illogicalapple",
      "link": "https://scratch.mit.edu/users/illogicalapple/"
    }
  ],
  "userscripts": [
    {
      "url": "userscript.js",
      "matches": ["https://scratch.mit.edu/projects/*"]
    }
  ],
  "settings": [
  	{
      "name": "Redirect to",
      "id": "redirectUrl",
      "type": "select",
      "default": "//turbowarp.org/",
      "potentialValues": [
        {
          "id": "//turbowarp.org/",
          "name": "TurboWarp"
```
(also in these places)